flags/1x1/my.svg

```diff
@@ -1,7 +1,18 @@
-<svg xmlns="http://www.w3.org/2000/svg" id="flag-icons-my" viewBox="0 0 512 512">
-  <path fill="#c00" d="M0 0h512v512H0z"/>
-  <path fill="#fff" d="M.4 42.5h511.2v41H.4zM.4 126.5h511.2v41H.4zM.4 210.6h511.2v41H.4z"/>
-  <path fill="#006" d="M0 0h256v298.7H0Z"/>
-  <path fill="#fc0" d="m172.5 86.8 5.6 38.2 21.6-32-11.6 36.8 33.3-19.4-26.4 28.1 38.5-3-36 13.8 36 14-38.5-3.1 26.4 28L188 169l11.6 36.7-21.6-31.9-5.6 38.1-5.6-38-21.5 31.8 11.5-36.7-33.2 19.4 26.3-28.1-38.4 3 36-13.9-36-13.9 38.4 3.1-26.3-28.1 33.2 19.4L145.4 93l21.5 32zm-31.2 1.6a66.7 66.7 0 1 0 0 121.9 75 75 0 1 1 0-121.9z"/>
-  <path fill="#fff" d="M.4 294.6h511.2v41H.4zM.4 378.1h511.2v41H.4zM0 471h511.2v41H0z"/>
+<svg viewBox="0 0 150 150" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The viewBox for 1x1 flags should be `0 0 512 512`. The flags all follow an identical viewBox and future flags/edits must adhere to this. Also, the id must be retained. Essentially, don't change the first line when not needed.